<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>550</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="DietLabel">
         <property name="text">
          <string>Dietary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="AllergiesLabel">
         <property name="text">
          <string>Allergies</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CaloriesLabel">
         <property name="text">
          <string>Calories</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TTCLabel">
         <property name="text">
          <string>Time to cook</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>200</number>
       </property>
       <property name="topMargin">
        <number>200</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FOODNAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IMAGE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Ingredients">
       <property name="text">
        <string>Ingredients: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Recipe">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuCreate_new">
     <property name="title">
      <string>Create new...</string>
     </property>
     <addaction name="actionAllergen"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuCreate_new"/>
    <addaction name="actionDelete_recipe"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuNew_Recipe"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDelete_recipe">
   <property name="text">
    <string>Delete recipe</string>
   </property>
  </action>
  <action name="actionAllergen">
   <property name="text">
    <string>Allergen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
